interpretation/polls.py

```diff
@@ -0,0 +1,166 @@
+from management.db import db_get, emoji_to_player, player_list, isParticipant
+from management.dynamic import day_number
+from interpretation.check import check_for_int
+
+class Vote:
+    def __init__(self,user_id,emoji,votes = 1):
+        if check_for_int(user_id) == True:
+            self.user = user_id
+            self.type = 'user'
+            self.emoji = emoji
+            self.votes = int(votes)
+        else:
+            self.user = user_id
+            self.type = 'generic'
+            self.emoji = emoji
+            self.votes = int(votes)
+
+class Disqualified:
+    def __init__(self,user_id,reason = 0):
+        self.user = user_id
+
+        self.reason = reason
+        # 0 - double voting
+        # 1 - self voting
+
+def count_votes(voting_table, purpose = 'lynch', mayor = 0):
+    """Count votes based on reactions given to """
+    user_table = []
+    blacklist = []
+    blacklist2 = []
+    emoji_table = []
+
+    for vote in voting_table:
+        # Add new emoji
+        if vote[1] not in emoji_table:
+            emoji_table.append(vote[1])
+
+        # Filter self-votes and double votes
+        if vote[0] in blacklist or vote[0] in blacklist2:
+            pass
+        elif vote[0] == emoji_to_player(vote[1]) and purpose not in ['Mayor','Reporter']:
+            blacklist2.append(vote[0])
+            if vote[0] in user_table:
+                user_table.remove(vote[0])
+        elif vote[0] in user_table:
+            blacklist.append(vote[0])
+            user_table.remove(vote[0])
+        else:
+            user_table.append(vote[0])
+
+    # Evaluate table, filter double votes and continue
+    voting_table = [Vote(vote[0],vote[1],int(db_get(vote[0],'votes'))) for vote in voting_table if vote[0] in user_table]
+    blacklist = [Disqualified(user) for user in blacklist]
+    blacklist2 = [Disqualified(user,1) for user in blacklist2]
+    blacklist.extend(blacklist2)
+
+    # Add extra (secret) outside lynches, may some special events happen.
+    if purpose == 'lynch':
+        for user in player_list():
+            if int(db_get(user,'threatened')) > 0:
+                print('{} has been threatened!'.format(user))
```

Comment:
Perhaps this should send to the gamelog?

Reply:
No, this is just for debugging. It doesn't actually threaten players, it tells the console it has recognized a threat. We can leave it out, I think.

Reply:
Can somebody remove this part?